lib/msal-browser/src/error/BrowserAuthError.ts

```diff
@@ -29,4 +33,9 @@ export class BrowserAuthError extends AuthError {
     static createNoWindowObjectError(): BrowserAuthError {
         return new BrowserAuthError(BrowserAuthErrorMessage.noWindowObjectError.code, BrowserAuthErrorMessage.noWindowObjectError.desc);
     }
+
+    static createPKCENotGeneratedError(arg0: string) {
```

Comment:
want to call this out as there are places where we have `PKCE`, `pkce`, and `Pkce`, we should probably normalize this, not only just for `PKCE` but also other acronyms

Reply:
I will try to make this uniform, thanks for the catch.

Reply:
Done.